src/Nethermind/Nethermind.Wallet/NodeKeyManager.cs

```diff
@@ -72,15 +72,14 @@ public PrivateKey LoadNodeKey()
             // this is not secure at all but this is just the node key, nothing critical so far, will use the key store here later and allow to manage by password when launching the node
             if (_config.TestNodeKey == null)
             {
-                var executingDirectory = PathUtils.GetExecutingDirectory();
-                string oldPath = Path.Combine(executingDirectory, UnsecuredNodeKeyFilePath);
-                string newPath = Path.Combine(executingDirectory, _config.KeyStoreDirectory, UnsecuredNodeKeyFilePath);
+                string oldPath = UnsecuredNodeKeyFilePath.GetApplicationResourcePath();
+                string newPath = Path.Combine(_config.KeyStoreDirectory.GetApplicationResourcePath(), UnsecuredNodeKeyFilePath);
```

Comment:
My intended usage was UnsecuredNodeKeyFilePath.GetApplicationResourcePath(_config.KeyStoreDirectory), but it may lead to same results, though abstracting everything in one place